VideoWeb/VideoWeb.EventHub/Hub/EventHubClient.cs

```diff
@@ -135,13 +140,14 @@ private async Task<string> GetObfuscatedUsernameAsync(string username)
         public async Task SendMessage(Guid conferenceId, string message)
         {
             var isAdmin = IsVhOfficerAsync();
+            var isJudge = IsJudgeAsync();
             var isAllowed = await IsAllowedToSendMessageAsync(conferenceId, isAdmin);
             if (!isAllowed) return;
             var from = Context.User.Identity.Name;
             var timestamp = DateTime.UtcNow;
 
             await Clients.Group(conferenceId.ToString())
-                .ReceiveMessage(conferenceId, from, message, timestamp, Guid.NewGuid());
+                .ReceiveMessage(conferenceId, from, message, timestamp, Guid.NewGuid(), isJudge);
```

Comment:
No neeed to extend the model here, not a suitable solution for when participant IM is enabled.

Reply:
done